package-build.el

```diff
@@ -153,6 +159,21 @@ function for access to this function")
   "Remove trailing whitespace from `STR'."
   (replace-regexp-in-string "[ \t\n]*$" "" str))
 
+
+
+(defun package-build--config (package)
```

Comment:
sorry i keep harping on this but this function should go away and the code should just live in the one place where it is used.